modules/vpc/main.tf

```diff
@@ -0,0 +1,27 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.63"
+
+  name = var.name
+  cidr = "10.${var.subnet[terraform.workspace]}.0.0/16"
```

Comment:
A more reusable module wouldn't be tied to the workspace directly, instead if its just passed in the value then I think it would be more versatile. Otherwise we can't have more than one of these per work space, which isn't something we're looking to do but it is something thats not hard to avoid.

Reply:
Ah good point. Removed the `[terraform.workspace]` portion and updated the var type.